draft-ietf-core-cf-reg-update.md

```diff
@@ -286,6 +288,10 @@ IANA is instructed to add the following note to the registry:
 > "Note: As per {{&SELF}}, temporary registrations within the 0-255 range are approved by Designated Experts.
 > These registrations are not subject to the formal {{-iana-early}} renewal process."
 
+## Reserving Content-Format Identifier 64999 for Documentation
+
+IANA is instructed to retire Content-Format identifier 64999 and mark it as reserved for use in documentation.
```

Comment:
Same comment as before: better not use "retire" here? We can "allocate" it or "reserve" it.